lib/drivers/tandemDriver.js

```diff
@@ -937,14 +968,25 @@ module.exports = function (config) {
             if (receive_seq != pkt.payload['header_log_seq_no']) {
               if (!recovering) {
                 recovering = true;
-                send_seq = receive_seq - 1;
                 debug('recovering ', receive_seq);
-                tandemCommand(COMMANDS.LOG_ENTRY_SEQ_REQ, [receive_seq], function (err) {
-                  if(err) {
-                    callback(err,null);
+              }
+
+              send_seq = receive_seq - 1;
+              tandemCommand(COMMANDS.LOG_ENTRY_SEQ_REQ, [receive_seq], function (err) {
+                if(err) {
+                  callback(err,null);
+                }
+                retryRecoverTimer = setTimeout(function() {
+                  if(recovering) {
+                    debug('Retrying to recover..',receive_seq);
+                    tandemCommand(COMMANDS.LOG_ENTRY_SEQ_REQ, [receive_seq], function (err) {
+                      if(err) {
+                        callback(err,null);
+                      }
+                    });
                   }
-                });
-              } // drop out-of-order packets on the floor; they will be re-requested.
+                },1000);
```

Comment:
I assume the timeout interval, would it be better to be a constant like the others?